dev/env/scripts/lib.sh

```diff
@@ -315,7 +318,21 @@ docker_logged_in() {
 }
 
 load_external_config() {
-  local chamber="${CHAMBER:-$CHAMBER_DEFAULT}"
+  local chamber
+  local use_aws_vault="${USE_AWS_VAULT:-$USE_AWS_VAULT_DEFAULT}"
+  if [ "$use_aws_vault" = true ]; then
+    local profile="${AWS_VAULT_PROFILE:-$AWS_VAULT_PROFILE_DEFAULT}"
+    local credentials_exist
+    credentials_exist=$(aws-vault list --credentials | grep "^${profile}$" || true)
+    if [[ ! $credentials_exist ]]; then
```

Comment:
Why not something like
```suggestion
    if [[ ! aws-vault list --credentials | grep -q "^${profile}$" ]]; then
```
?
Otherwise you are kind of throwing away the exit code (`|| true`) which exactly encodes this control boolean, no?

Reply:
`!` changed to `-z`. 
Here I'm checking the command output.
`|| true` is required so that no failure occurs when the pipe command (`grep`) returns a non-zero exit code.

Reply:
You are right. Fixed. Thanks for the detailed explanation.